Gruntfile.js

```diff
@@ -35,6 +38,24 @@ module.exports = function (grunt) {
           'src/',
         ],
       },
+      'build-out-wpt': {
+        cmd: 'node',
+        args: [
+          'node_modules/@babel/cli/bin/babel',
+          '--source-maps=false',
+          '--extensions=.ts',
+          '--delete-dir-on-start',
+          '--out-dir=out-wpt/',
+          '--only=src/common/framework/',
+          '--only=src/common/runtime/helper/',
+          '--only=src/common/runtime/wpt.ts',
+          '--only=src/webgpu/',
+          '--ignore=src/*/listing.ts',
```

Comment:
should this be ** ?
```suggestion
          '--ignore=src/**/listing.ts',
```

Reply:
No, we only have listings at the root of each suite. This needs to not match `src/common/framework/listing.ts` although that's incredibly unclear. I'll change this to just explicitly list the listing file since there's only one relevant one here.